docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,200 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'node-cluster-rsrc-use.json':
+      local legendLink = '%s/dashboard/file/k8s-node-rsrc-use.json' % $._config.grafana_prefix;
+
+      g.dashboard('USE Method / Cluster')
+      .addRow(
+        g.row('CPU')
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:avg_rate1m
+            *
+              instance:node_num_cpu:sum
+            / ignoring (instance) group_left
+              sum without (instance) (instance:node_num_cpu:sum)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          // TODO: Is this a useful panel?
+          g.panel('CPU Saturation (load1 per CPU)') +
+          g.queryPanel(|||
+            (
+              instance:node_load1_per_cpu:ratio
+            / ignoring (instance) group_left
+              count without (instance) (instance:node_load1_per_cpu:ratio)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          // TODO: Does `max: 1` make sense? The stack can go over 1 in high-load scenarios.
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+      )
+      .addRow(
+        g.row('Memory')
+        .addPanel(
+          g.panel('Memory Utilisation') +
+          g.queryPanel('instance:node_memory_utilisation:ratio', '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          g.panel('Memory Saturation (Swap I/O)') +
+          g.queryPanel('instance:node_memory_swap_io_bytes:sum_rate', '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes('Bps') },
+        )
+      )
+      .addRow(
+        g.row('Disk')
+        .addPanel(
+          g.panel('Disk IO Utilisation') +
+          // Full utilisation would be all disks on each node spending an average of
+          // 1 second per second doing I/O, normalize by metric cardinality for stacked charts.
```

Comment:
That doesn't sound right at all, it doesn't make sense to stack this as that's effectively summing it.

Reply:
Yes, that's why we divide by the number of metrics involved. If every disk is at full utilisation, it all adds up to 1 again.

Reply:
Modulo that it doesn't work if an instance has multiple devices.

I have added TODOs, but I guess that has to be solved right now as it is clearly not the intention of the original author to go above 1 here.

Reply:
With many disks in the cluster, un-stacked graphs will look messy. (Note that there is also a section in this dashboards where you look at individual instances.) I'll play with not aggregating aways the device and see how it works out.

Reply:
I have partitioned disk IO metrics by device. I think that could work. I'll play with it and tweak it later, but I think it's enough for the start we want to provide in this PR.

Reply:
I still think this is at best a very confusing graph. If you want to aggregate, max would be best.

Reply:
I want to check out how this graph works out in practice. I have added a TODO with the `max` idea (which I had contemplated before). I do not want to make the call in this PR.